pkg/kubelet/kubelet_getters.go

```diff
@@ -262,23 +262,23 @@ func (kl *Kubelet) GetPodCgroupRoot() string {
 	return kl.containerManager.GetPodCgroupRoot()
 }
 
-// GetHostIP returns host IP or nil in case of error.
-func (kl *Kubelet) GetHostIP() (net.IP, error) {
+// GetHostIPs returns host IPs or nil in case of error.
+func (kl *Kubelet) GetHostIPs() ([]net.IP, error) {
```

Comment:
Since callers of this always extract the IP at index 0, would it be better to keep this as GetHostIP and and return index 0 from the returned list at GetNodeHostIPs?

Reply:
I guess there's only one caller... I don't know, it seemed more appropriate to me to have the function return all the relevant information, and let the caller discard the secondary IP if it doesn't need it...

Reply:
Okay good point, missed that there is only 1 caller.